AOJ/03/03_C.py

```diff
@@ -0,0 +1,21 @@
+a = []
+while True:    
+    x,y = map(int,input().split())
+    if x == 0 and y == 0:
+        break
+    else:
+        a += [ (x,y) ]
+
+y = 0
+while True:    
+    if y == len(a):
+        break
+    else:
+        if a[y][0] < a[y][1]:
+            print(str(a[y][0]) + ' ' + str(a[y][1]))
```

Comment:
この条件文は、もし`y == len(a)` de`break`した場合、その後の処理は実行はキャンセルされます。そのため、以下のようにアンチパターンとして階層を浅くすることができます。
```suggestion
    if y == len(a):
        break
    if a[y][0] < a[y][1]:
        print(str(a[y][0]) + ' ' + str(a[y][1]))
```